pkg/plugin/connector/standalone/registry.go

```diff
@@ -60,30 +63,32 @@ func NewRegistry(logger log.CtxLogger, pluginDir string) *Registry {
 			r.logger.Warn(context.Background()).Err(err).Msg("could not extract absolute connector plugins path")
 		} else {
 			r.pluginDir = absPluginDir // store plugin dir for hot reloads
-			r.reloadPlugins()
```

Comment:
Moved into the `Init` method, so that we have two separate steps: one for just creating a registry and one for initializing it (loading all the plugins). This makes the registry more similar to other services constructed in runtime.go.